sdk/clojure/sdk-malli-schemas/build.clj

```diff
@@ -0,0 +1,49 @@
+(ns build
+  (:require [clojure.tools.build.api :as b]
+            [clojure.edn :as edn]))
+
+(def project (-> (edn/read-string (slurp "deps.edn"))
+                 :aliases :neil :project))
+(def lib (or (:name project) 'my/lib1))
+
+;; use neil project set version 1.2.0 to update the version in deps.edn
+
+(def version (or (:version project)
```

Comment:
I like to remove this `or` defaults on version and lib, they make no sense and if something goes wrong you don't want to release an artifact with those dummy values.

(Same for the other build.clj)

Reply:
You are right the neil generated default isn't that great. I can remove the the dummy artifact names. If no name is found I could exit 1 or just throw. Which is cleaner though? If there is an error here will the github workflow just fail and stop?

Reply:
Scratch my last comment, you already had setup the proper behavior in the other scripts. I replicated it.